spec/models/post_revision_spec.rb

```diff
@@ -6,6 +6,19 @@
   fab!(:user) { Fabricate(:user) }
   fab!(:post_revision) { Fabricate(:post_revision) }
 
+  fab!(:admin) do
+    user = Fabricate(:admin)
+    group = Group.find(Group::AUTO_GROUPS[:staff])
+    group.add(user)
+    user
+  end
+
+  before do
+    SiteSetting.set(:staff_alias_allowed_groups, Group::AUTO_GROUPS[:staff].to_s)
+    SiteSetting.set(:staff_alias_username, 'staff_alias_user')
+    SiteSetting.set(:staff_alias_enabled, true)
```

Comment:
Do you have tests that don't reset `enabled`?

Reply:
Good point, I've added a test case that changes `staff_alias_allowed_groups`.